### PR TITLE
Update Stem Creator URL

### PR DESCRIPTION
**Pull request by danielbayley**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/stemcreator.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version :latest
   sha256 :no_check
 
-  url 'http://stems-music.com/downloads/StemCreator_Installer.dmg'
+  url 'https://www.stems-music.com/download/stems-creator-installer-mac'
```
Comment (commitay) [inline Casks/stemcreator.rb]: The current URL from the `homepage` is `http://stemsmusic.wpengine.com/download/stems-creator-installer-mac/`

Comment (danielbayley) [reply Casks/stemcreator.rb]: @commitay They both trigger the same download. I thought the latter might require a comment for the slightly different domain… does it still need to be changed and said comment added?

Comment (commitay) [reply Casks/stemcreator.rb]: Yes, please change it and add the comment. They have the same download now but that may change in the future.

**Review by commitay — APPROVED** [review]

**Comment by danielbayley** [comment]
Ah I had done this offline but not pushed… this works.

**Pull request merged**